ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml

```diff
@@ -85,7 +84,8 @@ tests:
         ]
       REGION: us-west-2
       ROSA: "true"
-      SMCP_VERSION: "2.5"
+      SMCP_OPERATOR_VERSION: 2.6.0
```

Comment:
Is this needed?

Reply:
yes, if we merge some test specific for a newer operator, e.g. 2.6.1, it needs to be skipped in the lpinterop, e.g. : https://github.com/maistra/maistra-test-tool/blob/main/pkg/tests/non-dependant/olm_webhooks_test.go#L20